src/CodeGen_LLVM.cpp

```diff
@@ -281,8 +281,9 @@ CodeGen_LLVM *CodeGen_LLVM::new_for_target(const Target &target,
                         Target::Metal})) << "Zynq feature cannot be enabled with GPU features.\n";
         user_assert(target.arch == Target::ARM &&
```

Comment:
I think CodeGen_Zynq_LLVM/Zynq runtime might also work on X86, as apparently the C codegen works. So we might be able to remove the assert here, and use get_target_from_environment() as opposed to hardcode the target in stereo_hls/pipeline.cpp. This should solve the cross-compilation issue for setting up travis.